test/plugin_functional/plugins/kbn_tp_sample_panel_action/public/sample_panel_action.tsx

```diff
@@ -38,7 +38,8 @@ class SamplePanelAction extends ContextMenuAction {
     if (!embeddable) {
       return;
     }
-    openFlyout(
+    // TODO: Remove exclamation mark once we have a typed assurance core is there
+    getNewPlatform().start.core!.overlays.openFlyout(
```

Comment:
as I understand `start ---> setup` renaming is blocker for this PR.

probably adding exclamation mark manually is annoying and we can force type @elastic/kibana-platform 
```typescript
core: (null as unknown) as CoreSetup,
```

Reply:
As from past discussions with Court I think the suggestion was to basically check in `getNewPlatform()` is it already has been set and if not throw an error (since that could never happen) and thus then return a type that's not nullable.